routes/index.js

```diff
@@ -3,4 +3,4 @@ const router = Router();
 
 router.use("/calendars", require('./calendars'));
```

Comment:
```suggestion
router.use("/calendars", require('./calendars'));
router.use("/calendars/:calendarId/events", require('./events'));
```
Then you could move all those routes into a file like this.